docs/proposals/d-no-change.md

```diff
@@ -0,0 +1,114 @@
+# Proposal D: No Changes
+
+The following proposal describes how existing OCI components can be used to provide features of reference types without introducing new manifest schemas or distribution APIs.
+
+## Artifact Definition
+
+The artifact is defined using the existing OCI Artifact definition (image-spec to a custom config media type):
+
+```jsonc
+{
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "schemaVersion": 2,
+  "config": {
+    "mediaType": "<artifact-config-media-type>",
+    "size": 1234,
+    "digest": "sha256:a1a1a1..."
+  },
+  "layers": [
+    {
+      "mediaType": "<artifact-layer-media-type>",
+      "size": 1234,
+      "digest": "sha256:b2b2b2...",
+      "annotations": {
+        "com.example.metadata": "value"
+      }
+    }
+  ]
+}
+```
+
+Attaching an artifact to another object in the registry may be done with an OCI Index.
+For this example, the above artifact manifest is assumed to be `sha256:a0a0a0...`:
+
+```jsonc
+{
+  "mediaType": "application/vnd.oci.image.index.v1+json",
+  "schemaVersion": 2,
+  "manifests": [
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "digest": "sha256:010101...",
+      "size": 1234,
+      "platform": {
+        "architecture": "amd64",
+        "os": "linux"
+      }
+    },
+    { // the following is an artifact for the above linux/amd64 image
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "digest": "sha256:a0a0a0...",
+      "size": 1234,
+      "platform": {
+        "architecture": "amd64",
+        "os": "linux"
+      },
+      "annotations": {
+        "vnd.oci.artifact": "true", // this descriptor points to an artifact
```

Comment:
nit: Would it be enough to say that if a `artifact.type` is set, that it's an artifact? That would let us remove one more moving part in this proposal.

Reply:
My thought on this one is we only depend on the top level artifact = true field, and everything else is an optional field that can be used by clients to filter their future requests.